### DISCUSSION TITLE
job in data science

### DISCUSSION THREAD

**Discussion by Lal-babu21** [community/community#57591] [Programming Help]
### Select Topic Area

Question

### Body

Dear Readers,

I have been laid off for 6 months since I am unemployed. Any suggestions are welcome.

I possess data analytics, EDA, Visualization, Dashboard and Regression, classification, and clustering ML modeling skills.

Thank you
Lal babu

**Comment by mateobongiorno** [comment]
Hi Lal Babu.

Check out here: [Deep Learning AI](https://learn.deeplearning.ai/), and here:  [Google Cloud Skills](https://www.cloudskillsboost.google/journeys/118) do the courses, and update your LinkedIn and resume! 

It will help you.

Best!

**Comment by abdulmoiz-ds** [comment]
Hi Lal Babu, You have to polish your skills by doing [**Data Science Projects**](https://prostudytips.com/) and adding them to your portfolio. Then go to LinkedIn and show all the projects you have done so far. Companies will reach out to you.
You can practice the following Data Science Projects:
1. [Rainfall Prediction Using Python: Data Science Project](https://prostudytips.com/rainfall-prediction-using-python/)
2. [Face Recognition in Python – Data Analysis Projects](https://prostudytips.com/face-recognition/)
3. [Fake News Detection – Data Science Projects](https://prostudytips.com/fake-news-detection-data-science-projects/)
4. [Data Analysis Project – Airbnb Analysis EDA](https://prostudytips.com/data-analysis-project/)
5. [Twitter Sentiment Analysis in Python – Multinomial Naive Bayes](https://prostudytips.com/twitter-sentiment-analysis/)
6. [Stock Price Prediction in Python: How To – Data Science](https://prostudytips.com/stock-price-prediction-in-python/)
7. [Predict Disease with Machine Learning: How To – ProStudyTips](https://prostudytips.com/predict-disease-with-machine-learning/)
